Add FilterList size method to make it compatible with AsyncHBase

The size method returns the number of filters in the filter list. Since this isn't implemented by AsyncBigtable, it breaks the following test cases when building OpenTSDB:

../test/core/TestTsdbQueryQueries.java:1712: error: cannot find symbol
       assertEquals(2, filter_list.size());
                                  ^
   symbol:   method size()
   location: variable filter_list of type FilterList
 ../test/core/TestTsdbQueryQueries.java:1747: error: cannot find symbol
       assertEquals(2, filter_list.size());
                                  ^
   symbol:   method size()
   location: variable filter_list of type FilterList
 ../test/core/TestTsdbQueryQueries.java:1777: error: cannot find symbol
       assertEquals(2, filter_list.size());

--- a/src/main/java/org/hbase/async/FilterList.java
+++ b/src/main/java/org/hbase/async/FilterList.java
@@ -94,6 +94,13 @@ public final class FilterList extends ScanFilter {
     this.op = op;
   }
 
+  /** @return the number of filters in the filter list
+   *  @since 1.8 */
+  public int size() {
+    // filter's can't be null here and shouldn't even be empty
+    return filters.size();
+  }
+
   @Override
   byte[] name() {
     return NAME;

